_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Handbook of manners: how to behave in a code review"
```

Comment:
Please use [capital letters in headlines](http://www.grammarunderground.com/capitalization-of-headlines.html).
So this should be: "Handbook Of Manners: How To Behave In A Code Review".
Similar in all the other headlines in the article.